resources/guidelines/testing/store/quality-guidelines-plugins/README.md

```diff
@@ -0,0 +1,366 @@
+
+* [Code reviews for Shopware 6 on GitHub](https://github.com/shopwareLabs/store-plugin-codereview)
+
+### Automated code tests with Cypress
+
+There are Cypress tests for Shopware 6 on GitHub. The project is driven by the *Friends of Shopware* group. You can contribute at any time:
+
+* [Developer Documentation Cypress Tests for Shopware 6](https://developer.shopware.com/docs/guides/plugins/plugins/testing/end-to-end-testing)
+* [Cypress Tests for Shopware 6](https://github.com/shopware/platform/tree/trunk/src/Administration/Resources)
+
+### Helpful tools for app developers
+
+* [FroshPluginUploader](https://github.com/FriendsOfShopware/FroshPluginUploader): Tool for validating and uploading new SW6 app releases to the Community Store (GitHub Project from "Friends of Shopware")]
+* [Shopware CLI tools](https://github.com/shopwareLabs/sw-cli-tools): When you think about performance, these are various useful console helpers for generating data.
+
+### Descriptions in your Shopware account
+
+[App descriptions in your Shopware account](#app-descriptions-in-your-shopware-account) must follow the checklist criterion.
+
+## Automatic code review - Errors
+
+### The required composer.json file was not found
+
+**Cause:** Error in composer.json  
+
+One possible cause is that the technical app name from the Community Store or Account does not match the technical name entered in composer.json, or the app is incorrectly zipped. The technical app name has to be stored in the last part of the composer.json located at composore.json > extra > shopware-plugin-class. So take a look at the bootstrap class. Most of the errors is caused by the wrong technical name. For example, "Swag\\MyPlugin\\SwagMyPluginSW6" instead of "Swag\\MyPlugin\\SwagMyPlugin".
+
+[Here is an example](https://github.com/FriendsOfShopware/FroshPlatformPerformance/blob/master/composer.json#L20) of a valid composer.json.  
+See "[Plugin-Base Class](https://developers.shopware.com/designers-guide/javascript-statemanager-and-pluginbase/#plugin-base-class)" for more information.
+
+### Ensure cross-domain messages are sent to the intended domain
+
+See ["Cross-document messaging domains should be carefully restricted"](https://rules.sonarsource.com/javascript/RSPEC-2819) for more information.
+
+### No bootstrapping file found. Expecting bootstrapping in
+
+The bootstrap cannot be found. The reasons could be that the folder structure in the ZIP file is incorrect, there could be a typo, or a case-sensitive error in the app source (e.g., in the technical name).
+
+### Class Shopware\Storefront\* not found
+
+Missing requirements in the composer.json (e.g. "require": {"shopware/frontend": "*"},)  
+See "[Shopware App Development: App Meta Information - Explanation of the properties](https://docs.shopware.com/en/shopware-platform-dev-en/internals/plugins/plugin-information#explanation-of-the-properties)" for more information.
```

Comment:
This redirects to the home page pf dev docs. Please update this link.

Reply:
https://developer.shopware.com/docs/guides/plugins/plugins/plugin-base-guide#the-composer.json-file

This should be the correct link.